tests/std/include/parallel_algorithms_utilities.hpp

```diff
@@ -7,9 +7,13 @@
 #include <thread>
 
 #ifdef EXHAUSTIVE
-const size_t max_parallel_test_case_n = 1000;
+const size_t max_parallel_test_case_n = 4096;
+// // the number of linear steps a test case that is quadradic should attempt:

```

Comment:
```suggestion
// the number of linear steps a test case that is quadratic should attempt:
```
The variables are also typoed; I'll fix them.